lgc/interface/lgc/Pipeline.h

```diff
@@ -471,6 +471,9 @@ struct GeometryShaderMode {
   unsigned outputVertices;          // Max number of vertices the shader will emit in one invocation
 };
 
+// Kind of conservative depth
+enum class ConservativeDepth : unsigned { Any, Less, Greater };
```

Comment:
Actually, scratch that: could you please change the Less to LessEqual and the Greater to GreaterEqual? That's what those *really* mean semantically. It's unfortunate that for historical reasons, the SPIR-V modes are DepthGreater and DepthLess rather than being more explicit. HLSL does have explicit SV_DepthGreaterEqual and SV_DepthLessEqual.

Reply:
The reasoning here is that while the SPIR-V facing variables should of course mirror the names of the SPIR-V execution modes, we should be free to make the LGC definitions clearer as to their meaning.

Reply:
So are the hardware names EXPORT_LESS_THAN_Z and EXPORT_GREATER_THAN_Z also wrong?

Reply:
PS Thanks Kai for changing to a single variable that is an enum.

Reply:
Yes.

Reply:
Thanks, Nicolai. I've changed them in the latest pull request.